Source/WebKit/UIProcess/WebPageProxyInternals.h

```diff
@@ -345,6 +345,8 @@ struct WebPageProxy::Internals final : WebPopupMenuProxy::Client
 #if ENABLE(WRITING_TOOLS)
     HashMap<WTF::UUID, WebCore::TextIndicatorData> textIndicatorDataForAnimationID;
     HashMap<WTF::UUID, CompletionHandler<void(WebCore::TextAnimationRunMode)>> completionHandlerForAnimationID;
+    HashMap<WTF::UUID, CompletionHandler<void(std::optional<WebCore::TextIndicatorData>)>> completionHandlerForDestinationTextIndicatorForSourceID;
+    HashMap<WTF::UUID, WTF::UUID> sourceIDtoDestiationID;
```

Comment:
```suggestion
    HashMap<WTF::UUID, WTF::UUID> sourceIDtoDestinationID;
```

Reply:
May also want a name that has "animation" in it, as this is just a member of `WebPageProxyInternals`.